platform/linuxbsd/x11/display_server_x11.cpp

```diff
@@ -6062,10 +6043,24 @@ DisplayServerX11::DisplayServerX11(const String &p_rendering_driver, WindowMode
 			memdelete(rendering_context);
 			rendering_context = nullptr;
 			r_error = ERR_CANT_CREATE;
+
+			if (p_rendering_driver == "vulkan") {
+				String executable_name = OS::get_singleton()->get_executable_path().get_file();
+				OS::get_singleton()->alert(
+						vformat("Your video card drivers seem not to support the required Vulkan version.\n\n"
+								"If possible, consider updating your video card drivers or using the OpenGL 3 driver.\n\n"
+								"You can enable the OpenGL 3 driver by starting the engine from the\n"
+								"command line with the command:\n\n    \"%s\" --rendering-driver opengl3\n\n"
+								"If you recently updated your video card drivers, try rebooting.",
+								executable_name),
+						"Unable to initialize Vulkan video driver");
+			}
+
 			return;
```

Comment:
I notice that for other drivers we `ERR_FAIL_MSG`, but for Vulkan we print an error at the start with slightly different formatting, and then return. Should we unify those?

The same comment applies to Wayland if we do.